.mailmap

```diff
@@ -0,0 +1,5 @@
+Tomer Doron <tomerd@apple.com> <tomer.doron@gmail.com>
+Tomer Doron <tomerd@apple.com>
+Tomer Doron <tomerd@apple.com> tomer doron <tomer@apple.com>
```

Comment:
collapse?

Reply:
I think we can't collapse more as you used quite a few different spellings and emails

Reply:
that the way i roll